src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -9,7 +9,7 @@ description:
   The Notification Triggers API allows developers to schedule local notifications that don't require
   a network connection, which makes them an ideal solution for use cases like calendar apps.
 date: 2019-10-24
-updated: 2019-11-05
+updated: 2019-12-02
 draft: true
```

Comment:
```suggestion
```

@jpmedley & @tomayac should we also remove this `draft: true` flag so that the article gets published?

Reply:
Absolutely, but not before the actual OT goes live (which [hasn't happened yet](https://developers.chrome.com/origintrials/#/trials/active)). Beta is scheduled for [December 19](https://www.chromestatus.com/features/schedule).

Reply:
+1

Reply:
@tomayac & @jpmedley I'll leave it up to you to send a PR to remove `draft: true` when the timing is correct